auth/src/main/java/feast/auth/config/SecurityProperties.java

```diff
@@ -54,6 +53,7 @@
     private String provider;
 
     // K/V options to initialize the provider with
+    public static final String SUBJECT_CLAIM = "subjectClaim";
```

Comment:
Can we add a dedicated comment to this field?

Reply:
Added.